requirements-dev.txt

```diff
@@ -8,4 +8,8 @@ wheel
 jupyter
 pyzmq
 setuptools
-twine
\ No newline at end of file
+twine
+flake8
+clang-format==13.0.0
+mypy==0.782
+types-protobuf==3.18.4
```

Comment:
Shall we add a comment saying that the formatter versions here are corresponding to the versions in setup.py?

Reply:
Done